composer.json

```
@@ -27,8 +27,8 @@
     },
     "require-dev": {
         "atoum/atoum": "^2.8 || ^3.0",
```

To be compatible with PHP 7.x & 8.x, you shoud upgrade this package.

```suggestion
        "atoum/atoum": "^3.4 || ^4.0",
```

Maybe you will update unit tests

---

yup, this pr is only about following by requirements from upgraded versions of existed quality tools